### PR TITLE
CI stabilization

### PR DESCRIPTION
**Pull request by polybassa**
Add random trailer to filename to avoid permission errors on macOS

### CODE REVIEW

**Comment by polydroi** [comment]
@gpotter2 @guedou ready for review

File: scapy/asn1/mib.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
 _mib_re_integer = re.compile(r"^[0-9]+$")
 _mib_re_both = re.compile(r"^([a-zA-Z_][a-zA-Z0-9_-]*)\(([0-9]+)\)$")
 _mib_re_oiddecl = re.compile(
-    r"$\s*([a-zA-Z0-9_-]+)\s+OBJECT([^:\{\}]|\{[^:]+\})+::=\s*\{([^\}]+)\}", re.M)
+    r"$\s*([a-zA-Z0-9_-]+)\s+OBJECT[^:\{\}]+::=\s*\{([^\}]+)\}", re.M)
```
Comment (guedou) [inline scapy/asn1/mib.py:35]: Is there a unit test that covers this regexp? I am afraid that changing it might break things =/

**Review by guedou — APPROVED** [review]
LGTM. I made a small comment.

**Comment by polybassa** [comment]
Yes, there is a small test in regression.uts.

But honestly I don't know the entire scope of what should be parsed with this regex. What I can tell is, that previous there were three capture groups but only the first an the last are used in the later code.
-- 
Viele Grüße

Nils Weiß

On December 20, 2021 9:49:58 AM GMT+01:00, Guillaume Valadon ***@***.***> wrote:
***@***.*** approved this pull request.

**Comment by gpotter2** [comment]
LGTM, same comment than guedou.
The regex is used to parse som MIB files if I'm not mistaken(?)

**Comment by polybassa** [comment]
https://github.com/secdev/scapy/blob/65d4ba6e9b4f06518fb8320f06bfdba21802fd33/test/regression.uts#L3854

This checks the MIB parsing

**Review by gpotter2 — APPROVED** [review]

**Pull request merged**